src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -187,7 +188,7 @@ bool MulticopterLandDetector::_get_landed_state()
 		// falling consider it to be landed. This should even sustain
 		// quite acrobatic flight.
 		if ((_min_trust_start > 0) &&
-		    (hrt_elapsed_time(&_min_trust_start) > 8 * 1000 * 1000)) {
+		    (hrt_elapsed_time(&_min_trust_start) > 8e6)) {
```

Comment:
you are totally right, I only tried the literal alone
`(uint64_t) 8e6` would do the trick
not so nice again, sorry then